devices/climax.js

```diff
@@ -121,4 +121,17 @@ module.exports = [
         exposes: [e.battery_low(), e.tamper(), e.action(['emergency', 'panic', 'disarm', 'arm_all_zones', 'arm_day_zones']),
         ],
     },
+    {
+        zigbeeModel: ['WSS1_00.00.02.03TC'],
+        model: 'WSS1_00.00.02.03TC',
+        vendor: 'Climax',
+        description: 'Wall switch with 1 button',
+        fromZigbee: [fz.command_recall],
+        toZigbee: [],
+        exposes: [e.action('onOff')],
```

Comment:
Could you add all the send `action` values here? I expect something like: `e.action(['recall_1', 'recall_2']))`